server/src/test/java/org/elasticsearch/index/engine/InternalEngineTests.java

```diff
@@ -2367,6 +2368,7 @@ public void testConcurrentExternalVersioningOnPrimary() throws IOException, Inte
         }
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
I do not know if it is usually recommended to place it directly on top of the operation rather than on the whole function. Is the other suppression on 2477 needed?

Reply:
I think it's better to have the smallest scope for this annotation. I moved it on top of the operation in b628f625b422cbe33129684bf93895fad283bb52. Thanks for catching this!